index.js

```diff
@@ -129,6 +131,14 @@ const parseArgs = ({
     }
   );
 
+  const checkSafeOptionValue = (dashedOption, longOption, value) => {
+    if (strict && !isSafeOptionValue(value)) {
+      const errorMessage = `Did you forget to specify the option argument for '${dashedOption}'?
+To specify an option argument starting with a dash use '--${longOption}=-EXAMPLE'.`;
+      throw new Error(errorMessage);
```

Comment:
we should add this to `errors.js`.

Reply:
Or (now it exists) could use `ERR_PARSE_ARGS_INVALID_OPTION_VALUE` which takes a freeform string?

Reply:
Switched to `ERR_PARSE_ARGS_INVALID_OPTION_VALUE`